docs/installation.md

```diff
@@ -0,0 +1,243 @@
+# Installing ansible-navigator with execution environment support
+
+[podman_url]: https://podman.io/
+
+```{contents}
+:local:
```

Comment:
Side note: I wish we could just use the Furo theme. Then, we wouldn't need the local ToC at all.